app/console.go

```diff
@@ -2,19 +2,29 @@ package app
 
 import (
 	"strconv"
+	"sync"
 
 	"github.com/abiosoft/ishell"
 	"github.com/ubclaunchpad/cumulus/blockchain"
-	"github.com/ubclaunchpad/cumulus/conn"
 	"github.com/ubclaunchpad/cumulus/peer"
 )
 
-var shell *ishell.Shell
+var (
+	shell     *ishell.Shell
+	peerStore *peer.PeerStore
```

Comment:
Agreed, I see what you're doing, but it is a strange place to have these globals floating around.